<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedBSAModule</class>
 <widget class="QWidget" name="AdvancedBSAModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="baseGroupBox">
     <property name="title">
      <string>BSA</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="BSAExtract">
        <property name="text">
         <string>Extract BSAs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="BSACreate">
        <property name="text">
         <string>Create BSAs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="BSAProcessContent">
        <property name="text">
         <string>Optimize BSAs content</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="moreGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>More</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1">
       <widget class="QCheckBox" name="bsaCreateDummiesCheckbox">
        <property name="text">
         <string>Create dummy plugins</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="bsaLeastBsaCheckBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;There are three types of BSAs CAO can create : uncompressible, textures and standard. The uncompressible one contains sounds. If this option is checked, these three types of BSAs will be merged if possible.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Create the least BSAs possible</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="allowCompression">
        <property name="text">
         <string>Allow BSA compression</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="advancedGroupBox">
     <property name="title">
      <string>Advanced</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="maxStandardSize"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="createTexturesBSAGroupBox">
        <property name="title">
         <string>Create textures BSA</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="maxTexturesSize"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Uncompressed max size</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Uncompressed max size</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
